kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -54,26 +80,166 @@
           this.isFullscreen = true;
         }
       },
+      getPage(pageNum) {
+        return this.pdfDocument.getPage(pageNum);
+      },
+      displayPage(pdfPage) {
+        return new Promise((resolve, reject) => {
+          const pageNum = pdfPage.pageNumber;
+          if (this.pdfPages[pageNum]) {
```

Comment:
If this is undefined, the whole thing has been tidied up before we got here, forget about doing anything else.